src/main/java/com/epam/ta/reportportal/core/item/impl/UpdateTestItemHandlerImpl.java

```diff
@@ -288,6 +291,61 @@ public void resetItemsIssue(List<Long> itemIds, Long projectId) {
 		});
 	}
 
+	@Override
+	public OperationCompletionRS bulkUpdateAttributes(BulkUpdateItemAttributeRQ bulkUpdateRq,
```

Comment:
maybe we can try to unify all this logic with launches by one common utility class.